redirects.yaml

```diff
@@ -623,5 +623,6 @@ wiki.*/?: "https://wiki.ubuntu.com/"
 yourcloud/?: "/cloud/managed-cloud"
 ziskejte/prechod910/?: "/download"
 certification(?P<page>/.*)?: "https://certification.ubuntu.com{page}"
+usn/USN-(?P<page>[^/]+)/?: "https://usn.ubuntu.com/{page}"
```

Comment:
Could you also add a redirect for `USN-*` (without the `usn/` prefix)? Because by this time some people will presumably have followed and cached redirects to 404-ing URLs like https://usn.ubuntu.com/USN-3593-1

Reply:
Sure, should be fine to add. I can make the usn prefix optional on this rule.

Reply:
The optional part didn't seem to work so easily, I added a new line